docs/csharp/language-reference/builtin-types/reference-types.md

```diff
@@ -31,7 +31,7 @@ The `object` type is an alias for <xref:System.Object?displayProperty=nameWithTy
 
 ## The string type
 
-The `string` type represents a sequence of zero or more Unicode characters. `string` is an alias for <xref:System.String?displayProperty=nameWithType> in .NET.
+The `string` type represents a sequence of zero or more Unicode characters. `string` is an alias for <xref:System.String?displayProperty=nameWithType> in .NET. If you have `using System;` in your file, `string` and `String` are equivalent.
```

Comment:
Even if `using System;` is there. They may not be equivalent (https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html):

```csharp
using System;

String x = "Hello, world!";
Console.WriteLine(x);

namespace System
{
    class String
    {
        public static implicit operator String(string s) => null;
    }
}
```

It's a corner case that I may *not* be worth mentioning in docs.

Reply:
I agree with @Youssef1313 here on this, and thank you for sharing @jaredpar post -- it is a very good concern to be aware of.

Reply:
```suggestion
The `string` type represents a sequence of zero or more Unicode characters. `string` is an alias for <xref:System.String?displayProperty=nameWithType> in .NET.
```

I do not believe that this is accurate to say, nor do I think it is needed here.